lib/fathead/jmeter/fetch.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Can you please rename the directory `jmeter_docs/` instead of `jmeter/`. This might be the reason @daxtheduck is failing to push to the dev machine.